### PR TITLE
Add GitLab CI example

### PR DESCRIPTION
**Pull request by ikalinin1**

### CODE REVIEW

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+image: openjdk:17.0.2-slim
+stages:
+  - build
+  - test
+
+variables:
+  OUT: .out
+  HISTORY: $OUT/$CI_PROJECT_NAME/history
```
Comment (valfirst) [inline .gitlab-ci.yml:8]: what is `$CI_PROJECT_NAME`?

File: .gitlab-ci.yml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  script:
+    - ./gradlew runStories -Pvividus.configuration.suites=rest_api -Pvividus.allure.history-directory=$HISTORY/rest-api-tests
+  cache:
+    key: "$CI_JOB_NAME-$CI_JOB_STAGE"
+    paths:
+       - $HISTORY
+    when: always
+  artifacts:
+    when: always
+    name: API Allure Report
```
Comment (valfirst) [inline .gitlab-ci.yml:39]:
```suggestion
    name: REST API tests report
```

File: .gitlab-ci.yml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                           -Pvividus.selenium.grid.capabilities.browserName=chrome \
+                           -Pvividus.allure.history-directory=$HISTORY/web-app-tests
+  cache:
+    key: "$CI_JOB_NAME-$CI_JOB_STAGE"
+    paths:
+       - $HISTORY
+    when: always
+  artifacts:
+    when: always
+    name: Chrome Allure Report
```
Comment (valfirst) [inline .gitlab-ci.yml:64]:
```suggestion
    name: Web application tests report
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: .gitlab-ci.yml (hunk 1 above)
Comment (ikalinin1) [reply .gitlab-ci.yml:8]: https://docs.gitlab.com/ee/ci/variables/#list-all-environment-variables

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**